app/controllers/service_providers_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ServiceProvidersController < AuthenticatedController
-  before_action :authorize_service_provider
+  before_action -> { authorize ServiceProvider }, only: [:index, :create, :new, :all]
+  before_action -> {
+ authorize service_provider, :member_or_admin? }, only: %i[update edit show destroy]
```

Comment:
spacing got funky here, maybe realign like this? also I'd add parens for clarity
```suggestion
  before_action -> { authorize(service_provider, :member_or_admin?) },
                only: %i[update edit show destroy]
```

Reply:
hahaha this is what rubocop did on its own. bad linter!

Reply:
fixed as part of https://github.com/18F/identity-dashboard/pull/770/commits/17dc2014e97cfbfff15583686ae23b9fbadd8b30